### PR TITLE
OpenMC/CG summary property getters

### PR DESCRIPTION
**Pull request by wbinventor**
This PR removes the `Summary.make_opencg_geometry()` routine from the Python API. Now one can simply use the `Summary.opencg_geometry` property getter which will construct an OpenCG geometry (if one has OpenCG installed of course). Similarly, a new `Summary.openmc_geometry` property getter has been added.

In addition, a new `Tally.__div__(self, other)` routine has been added which simply defers to the existing `Tally.__truediv__(self, other)`. I realized that the "tally-arithmetic" ipython notebook was failing since PR #471 for Python 2 since I moved to `__truediv__` for Python 3 compatibility with tally arithmetic. Although "tallies.py" uses `from __future__ import division`, the tally division in the "tally-arithmetic" ipython notebook was not calling `__truediv__`. This PR fixes that issue for both Python 2 and 3.

### CODE REVIEW

File: openmc/summary.py [hunk 1]
```diff
@@ -23,13 +32,30 @@ def __init__(self, filename):
...
+    @property
+    def openmc_geometry(self):
+        return self._openmc_geometry
+
+    @property
+    def opencg_geometry(self):
+        try:
+            from openmc.opencg_compatible import get_opencg_geometry
+        except ImportError:
+            msg = 'Unable to import OpenCG.'
```
Comment (paulromano) [inline openmc/summary.py]: Without the try/except, wouldn't you get an almost equivalent message (No module named opencg)?

Comment (wbinventor) [inline openmc/summary.py]: Good point, I removed the `try-catch`.

File: openmc/summary.py [hunk 2]
```diff
@@ -468,7 +488,7 @@ def _finalize_geometry(self):
 
         # Set the root universe for the Geometry
         root_universe = self.get_universe_by_id(0)
-        self.openmc_geometry.root_universe = root_universe
+        self._openmc_geometry.root_universe = root_universe
```
Comment (paulromano) [inline openmc/summary.py]: Why not just use the property?

Comment (wbinventor) [inline openmc/summary.py]: Good catch, it uses the property now

**Comment by wbinventor** [comment]
I just pushed a fix to not only the issued you raised @paulromano, but also with a new `opencg_property` for the `openmc.mgxs.Library` class. This property mirrors the one for the `Summary` class. If you'd rather not see this introduced in this PR, that's fine, I'll remove it.

**Comment by paulromano** [comment]
Thanks @wbinventor!

**Pull request merged**